src/Links.elm

```diff
@@ -60,6 +60,12 @@ link name =
         Ffmprovisr ->
             a [ href "https://amiaopensource.github.io/ffmprovisr/" ] [ text "ffmprovisr" ]
 
+        HessReel ->
+            a [ href "http://richardhess.com/notes/formats/magnetic-media/magnetic-tapes/analog-audio/025-reel-tape" ] [ text "0.25\" reel tape" ]
+
+        HessTimingChart ->
+            a [ href "http://richardhess.com/notes/formats/magnetic-media/magnetic-tapes/analog-audio/tape-timing-chart/" ] [ text "Tape Timing Chart" ]
+
```

Comment:
You can use both times `https` rather than `http`.

Reply:
Updated! Thank you @retokromer